emscripten.py

```diff
@@ -1895,8 +1894,15 @@ def debug_copy(src, dst):
       '--entry=main',
       '--allow-undefined',
       '--import-memory',
-      '--export', '__wasm_call_ctors',
-    ])
+      '--export', '__wasm_call_ctors']
+    for export in shared.expand_response(settings['EXPORTED_FUNCTIONS']):
+      cmd.append('--export')
+      cmd.append(export[1:]) # Strip the leading underscore
+    shared.check_call(cmd)
+
+    if DEBUG:
+      logging.debug('  emscript: lld took %s seconds' % (time.time() - t))
```

Comment:
Why did this move?

Reply:
So it reports the actually time that lld took, and not the other stuff we are doing here.

Reply:
Ah right, we split this out from the wasm backend log earlier. I got confused at some point and thought it was timing all of `build_wasm_lld`